src/GMB.jl

```diff
@@ -0,0 +1,71 @@
+struct GMB{S}
+    filepath :: Vector{S}
+end
+
+function GMB(dirpath)
+    @assert(isdir(dirpath), dirpath)
+        paths=String[]
+            data_path = joinpath.(dirpath,"data")
+                for dir in readdir(data_path)
+                    for d in readdir(joinpath.(data_path,dir))
+                         if ("en.tags" in readdir(joinpath.(data_path,dir,d))) == true
+                          push!(paths,joinpath.(data_path,dir,d,"en.tags"))
+                         end
+                       
+                    end
+                end
+  
+GMB(paths)
+
+end
+
+GMB() = GMB(datadep"GMB 2.2.0") 
+
+MultiResolutionIterators.levelname_map(::Type{GMB}) = [
+    :doc=>1, :contextfile=>1, :context=>1, :document=>1,
+    :para=>2, :paragraph=>2,
+    :sent=>3, :sentence=>3,
+    :word=>4, :token=>4,
+    :char=>5, :character=>5
+    ]
+
+function parse_gmb_tagged_word(line::AbstractString)
+    tokens_tags = split(line,"	")
```

Comment:
Would `\t` would be clearer because it doesn't look like a bunch of spaces? (also fixing spacing)
```suggestion
    tokens_tags = split(line, '\t')
```